database.sql

```diff
@@ -1076,6 +1076,16 @@ CREATE TABLE IF NOT EXISTS `userd` (
 	 INDEX `username` (`username`(32))
 ) DEFAULT COLLATE utf8mb4_general_ci;
 
+--
+-- TABLE user-item
+--
+CREATE TABLE IF NOT EXISTS `user-item` (
+	`iid` int unsigned NOT NULL DEFAULT 0 COMMENT 'Item id',
+	`uid` mediumint unsigned NOT NULL DEFAULT 0 COMMENT 'User id',
+	`hidden` boolean NOT NULL DEFAULT '0' COMMENT 'Hidden marker',
```

Comment:
Can you make the comment more specific? Where is the item hidden from?

Reply:
I'm unsure how to describe it.

Reply:
"Hides a public item from the user view"?